packages/expo-notifications/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### 🎉 New features
 
+- warn about usage with Expo Go ([#31573](https://github.com/expo/expo/pull/31573) by [@vonovak](https://github.com/vonovak))
```

Comment:
It's not a new feature 😅 Let's move it to `Others` category and maybe rephrase to something like
```suggestion
- Warn about limited support in Expo Go ([#31573](https://github.com/expo/expo/pull/31573) by [@vonovak](https://github.com/vonovak))
```
which seems to be more meaningful